pkg/controller/acicontainersoperator.go

```diff
@@ -413,7 +546,7 @@ func (c *Controller) CreateAciContainersOperatorObj() *operators.AciContainersOp
 
 func (c *Controller) CreateAciContainersOperatorCR() error {
 	log.Info("Reading the aci-operator-config Config Map providing CR")
-	rawSpec, err := c.ReadConfigMap("aci-containers/aci-operator.conf")
+	rawSpec, err := c.ReadConfigMap("/usr/local/etc/aci-containers/aci-operator.conf")
```

Comment:
Here and elsewhere below - why is this change needed? Making the file path absolute would mean that we have to be really sure that it remains the same in all situations. Would that be the case? And even if that were to be the case, this prefix should probably be pre-defined somewhere else and used here.